Header header

float32 speed
float32 front_left_angle
float32 front_right_angle
float32 back_left_angle
float32 back_right_angle
float32 total_amps
float32 battery_pct
float32 max_drive_temp
int8    hottest_motor_id
float32 roll
float32 pitch
float32 heading

